modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -60,6 +60,20 @@ Review any installed third-party apps for compatibility with the new ownCloud re
 Ensure that they are all disabled before beginning the upgrade.
 Third party apps are all apps that are not distributed by https://marketplace.owncloud.com/publishers/owncloud[ownCloud]
 
+[TIP]
```

Comment:
What happened here?
This commit has nothing to do with the topic of this PR.

Reply:
Odd. I'll get it removed.

Reply:
On reviewing it further, I have no idea why I committed that with this PR.